draft-patton-cfrg-vdaf.md

```diff
@@ -176,6 +176,37 @@ is usually handled by raising an exception.
 
 # Distributed Aggregation Functions {#daf}
 
+<!--
+
+RLB: We need to elaborate the operational model a bit more here.  Maybe in a
+protocol overview section.  It seems like the major roles / steps are:
+
+* Roles: 
+  * Submitter - Knows raw metrics
+  * Aggregator - Trusted not to collued
+  * Collector - Receives aggregate at the end of the protocol (may ==
+    Aggregator)
+* Measurement process (covering both DAF and VDAF):
+  0. Raw measurement - Submitter has a measurement to submit
+  1. Sharding - Submitter computes input shares from raw measurement value
+  2. Validation - Aggregators determine that the input shares are valid
+  3. Aggregation - Aggregators compute agregate shares from input shares
+  4. Unsharding - Aggregators / Collector commpute aggregate output from
+     aggregate shares
+
+Matching the below to this framework, I see a few issues:
+
+a. The diagram indicates that the output shares are sent to aggregators
+b. The meaning of the `param` input is unclear.  Is that the working aggregate?
```

Comment:
The aggregation parameter is what maps an input to an output. For heavy hitters for example, the aggregation parameter is the set of candidate prefixes. I'll make this more clear.